state_test.go

```diff
@@ -862,13 +862,26 @@ func TestMemberList_ResetNodes(t *testing.T) {
 	d := dead{Node: "test2", Incarnation: 1}
 	m.deadNode(&d)
 
+	oldGossipToTheDeadTime := m.config.GossipToTheDeadTime
+	m.config.GossipToTheDeadTime = 3 * time.Second
```

Comment:
Let's set this to something like `200 * time.Millisecond` so the test doesn't take too long.